File: generated_codes/experiment_b/java_files/28.java
/*
*/


import java.util.*;
public class Hello {

    public static void main(String[] args) {
		//Your Code Here
		Scanner sc=new Scanner(System.in);
		int n=sc.nextInt();
		while(n-- >0){
	        char c=sc.next().charAt(0);
	        switch(c){
	            case 'B':
	            case 'b':
	                System.out.println("BattleShip");
	                break;
	            case 'C':
	            case 'c':
	                System.out.println("Cruiser");
	                break;
	            case 'D':
	            case 'd':
	                System.out.println("Destroyer");
	                break;
	            case 'F':
	            case 'f':
	                System.out.println("Frigate");
	                break;
	            default:
	                break;
	        }
		}
	}
}

File: generated_codes/experiment_c/parameter_set_2/single_sample/python_files/28.py
'''

'''

import sys

def ship_class(c):
    if c == 'B' or c == 'b':
        return 'BattleShip'
    elif c == 'C' or c == 'c':
        return 'Cruiser'
    elif c == 'D' or c == 'd':
        return 'Destroyer'
    elif c == 'F' or c == 'f':
        return 'Frigate'
    else:
        return 'Error'

t = int(input())
for i in range(t):
    c = input()
    print(ship_class(c))